pwndbg/commands/xor.py

```diff
@@ -31,10 +31,17 @@ def xor(address, key, count):
     XOR ``count`` bytes at ``address`` with the key ``key``.
     '''
     if not isinstance(address, six.integer_types):
-        address = int(address, 16)
+        try:
+            address = int(address, 0)
+        except ValueError:
+            print('Invalid address %s' % address)
+            return
 
-    xorred_memory = xor_memory(address, key, count)
-    pwndbg.memory.write(address, xorred_memory)
+    try:
+        xorred_memory = xor_memory(address, key, count)
+        pwndbg.memory.write(address, xorred_memory)
+    except gdb.MemoryError as e:
+        print(e)
 
 @pwndbg.commands.Command
```

Comment:
its just a string. lets say i want to xor it with "FOOBAR" its quite handy to xor with a ascii xor key instead of only bein able to pass bytes via 0x42424242